src/harness/sas.py

```diff
@@ -333,6 +332,10 @@ def TriggerDpaActivation(self, request):
   def TriggerDpaDeactivation(self, request):
     _RequestPost('https://%s/admin/trigger/dpa_deactivation' %
                  self._base_url, request, self._tls_config)
+    
+  def TriggerFullActivityDump(self):
```

Comment:
Seems like there might be a small merge issue here -- these lines are duplicates of the following lines.